src/Discord.Net.Commands/SlashCommands/Attributes/CommandGroup.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Discord.SlashCommands
+{
+    /// <summary>
+    ///     Defines the current as being a group of slash commands.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+    public class CommandGroup : Attribute
+    {
+        /// <summary>
+        ///     The name of this slash command.
+        /// </summary>
+        public string groupName;
+
+        /// <summary>
+        ///     The description of this slash command.
+        /// </summary>
+        public string description;
```

Comment:
Same principle here:

```suggestion
        /// <summary>
        ///     The name of this slash command.
        /// </summary>
        public string GroupName { get; set; }

        /// <summary>
        ///     The description of this slash command.
        /// </summary>
        public string Description { get; set; }
```